lib/node_modules/@stdlib/math/base/special/factorial2/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env julia
```

Comment:
Might be better to use Python and SciPy for generating test fixtures, as SciPy actually has a `factorial2` implementation against which we can test.